go/mysql/query_test.go

```diff
@@ -128,7 +128,8 @@ func TestComStmtPrepare(t *testing.T) {
 		cConn.Close()
 	}()
 
-	sql := "select * from test_table where id = ?"
+	sql := "select ?; select ?"
+	//sql := "select * from test_table where id = ?"
```

Comment:
Looks like this is just a debugging/testing change? Should this test fail when multiple statements are included? Seems like it's still passing though?

Reply:
Oops yeah, this was a debugging change.
This test should fail when there are multiple statments; it doesn't currently because the check is in `handleNextCommand`.

I'll move this test case to a different testing suite, and check for an error.